errors.js

```diff
@@ -1,21 +1,26 @@
 const utError = require('ut-error');
-const create = utError.define;
-const Bus = create('bus');
-const UnhandledError = create('unhandledError', Bus);
 
-module.exports = {
-    bus: Bus,
-    defineError: create,
-    getError: utError.get,
-    timeout: create('timeout', Bus, 'Time out'),
-    methodNotFound: create('methodNotFound', Bus, 'Method {method} not found'),
-    destinationNotFound: create('destinationNotFound', Bus, 'Destination {destination} not found'),
-    missingMethod: create('missingMethod', Bus),
-    unhandledError: function($meta) {
-        var err = new UnhandledError('Unhandled Error' + ($meta.errorMessage ? ': ' + $meta.errorMessage : ''));
-        if ($meta.errorCode) {
-            err.code = $meta.errorCode;
+module.exports = (bus) => {
+    utError.init(bus);
```

Comment:
If you call this unconditionally, then you need to update the ut-error version in package.json.
You can then push this in minor/rc-cubalibre and we will have the branch.

Reply:
ut-error have had the init for a long time. It was just not used: https://github.com/softwaregroup-bg/ut-error/blob/master/index.js#L72